docs/pywbemclicmdshelp.rst

```diff
@@ -191,9 +191,9 @@ The following defines the help output for the `pywbemcli class associators --hel
       -c, --resultclass <class name>  Filter by the result class name provided.
       -r, --role <role name>          Filter by the role name provided.
       -R, --resultrole <role name>    Filter by the role name provided.
-      --includequalifiers / --no_includequalifiers
-                                      Include qualifiers in the result. Default is
-                                      to include qualifiers
+      --no-qualifiers                 Do not include qualifiers in the
+                                      response.The default behavior is to
+                                      includequalifiers in the returned class.
```

Comment:
Insert a space in "includequalifiers". Occurs multiple times.

Reply:
DONE. Think there was only one but it shows up in the help and tests also.